__tests__/app.test.js

```diff
@@ -220,3 +221,39 @@ describe("GET/api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("POST /api/articles/:article_id/comments", () => {
+  test("returns status:201 and POST comment retruns all the properties of the posted object", () => {
+    const addComment = {
+      author: "rogersop",
+      body: "testComment",
+    };
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send(addComment)
+      .expect(201)
+      .then((res) => {
+        const { comment } = res.body;
+
+        expect(comment).toHaveProperty("comment_id", expect.any(Number));
+        expect(comment).toHaveProperty("article_id", 1);
+        expect(comment).toHaveProperty("author", expect.any(String));
+        expect(comment).toHaveProperty("body", expect.any(String));
+        expect(comment).toHaveProperty("votes", expect.any(Number));
+        expect(comment).toHaveProperty("created_at", expect.any(String));
+      });
+  });
+  test("returns status:400 and returns error message when posting comment to invalid article_id", () => {
+    const addComment = {
+      author: "rogersop",
+      body: "testComment",
+    };
+    return request(app)
+      .post("/api/articles/100/comments")
+      .expect(400)
+      .then((res) => {
+        const { msg } = res.body;
+        expect(msg).toBe("Bad request");
+      });
+  });
```

Comment:
Can we consider any more errors here?
- Problems with the input
- 404 article not found (tweaking the above test)
- 400 invalid article_id